packages/mock-app/src/components/CredentialTabs/CredentialTabs.tsx

```diff
@@ -43,12 +47,44 @@ const CredentialTabs = ({ credential }: { credential: VerifiableCredential }) =>
 
   return (
     <Box sx={{ width: '100%', bgcolor: 'background.paper', minHeight: '300px' }}>
-      <Tabs value={currentTabIndex} onChange={handleChange} centered>
-        {credentialTabs?.map((item, index) => <Tab key={index} label={item.label} />)}
-      </Tabs>
+      {/* Header Row */}
+      <Box
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+          flexDirection: isMobile ? 'row' : 'row',
```

Comment:
Could you double check this line of code? Is it correct?

Reply:
Fixed, thanks.